processor/src/test/java/org/mapstruct/ap/test/nestedbeans/NestedSimpleBeansMappingTest.java

```diff
@@ -41,6 +44,13 @@
 @RunWith(AnnotationProcessorTestRunner.class)
 public class NestedSimpleBeansMappingTest {
 
+    @Rule
+    public final GeneratedSource generatedSource = new GeneratedSource().addComparisonToFixtureFor(
```

Comment:
There doesn't seem to be an actual test for the generated functionality? Having the assertion on the generated source is great, but there should be a test the actual functionality, too.